dss/events/handlers/sync.py

```diff
@@ -62,6 +62,10 @@ def sync_s3_to_gs_oneshot(source: BlobLocation, dest: BlobLocation):
         ExpiresIn=presigned_url_lifetime_seconds
     )
     with closing(http.request("GET", s3_blob_url, preload_content=False)) as fh:
+        if 200 != fh.status:
+            msg = f"failed to resolve s3 presigned url for {source.blob.key}"
+            logger.error(msg)
+            raise Exception(msg)
```

Comment:
We need a more specific exception. How about `DSSException`?

Reply:
look at https://github.com/HumanCellAtlas/data-store/blob/master/dss/error.py for ideas